### ISSUE TITLE
Animation FPS and SCALE tag

### ISSUE THREAD

**Issue by enzyme69** [kcoley/gltf2usd#24]
I wonder at what format I should specify the animation FPS and Scale? I tried 0.1 for scaling and giving me error. It needs INT... somewhat.

FPS also... I tried 30 for example:

```
  File "gltf2usd.py", line 1126, in <module>
    convert_to_usd(args.gltf_file, args.usd_file, args.fps, args.scale, args.verbose)
  File "gltf2usd.py", line 1114, in convert_to_usd
    GLTF2USD(gltf_file=gltf_file, usd_file=usd_file, fps=fps, scale=scale, verbose=verbose)
  File "gltf2usd.py", line 70, in __init__
    self.convert()
  File "gltf2usd.py", line 1100, in convert
    self.convert_nodes_to_xform()
  File "gltf2usd.py", line 106, in convert_nodes_to_xform
    self._convert_animations_to_usd()
  File "gltf2usd.py", line 516, in _convert_animations_to_usd
    self._convert_skin_animations_to_usd()
  File "gltf2usd.py", line 608, in _convert_skin_animations_to_usd
    max_time = int(round(input_keyframe_accessor['max'][0] * self.fps))
TypeError: can't multiply sequence by non-int of type 'float'
```

**Comment by ox** [comment]
this was actually closed by #26

**Comment by kcoley** [comment]
@ox thanks for the PR.  I also updated scale to take a float as well, so this should resolve this issue.
